spec/factories/genres.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :genre do
+    name "My name"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.